### ISSUE TITLE
What name / version / branch to use in package.json dependencies?

### ISSUE THREAD

**Issue by desmond-dsouza** [callstackincubator/rebolt-navigation#112]
A bit confused about changes of name / dependencies in what was `reroute-native`.

Where I previously had: `"reroute-native": "github:knowbody/reroute-native#master" `, what name / version / branch should I now use for a new project? And do I drop `bs-react-native` now? Is there currently a working clone-able example (or instructions) to start?

Thanks!

**Comment by knowbody** [comment]
Sorry for the confusion, we are in the process of renaming reroute-native to rebolt-navigation. So, for now, use the latest version of reroute-native and we will be releasing the next version as rebolt-navigation with bumped up version, so later you will change the name of the package and bump up the version.

You can still use bs-react-native, but we will recommend switching to rebolt instead as those two will start having a closer dependency on one another very soon.

Bear with me and I will try to update the docs here and provide an example. cc @grabbou

**Comment by grabbou** [comment]
Yup, we are planning next release this week once we move all the internal
bindings within `rebolt-navigation` to `rebolt` package. This is on my todo
list for tomorrow.
